R/code_accession.R

```diff
@@ -0,0 +1,130 @@
+#' Get conditions and processes to accede memberships
+#'
+#' The function allows to get information on the conditions
+#' for parties to enter a treaty.
+#' It could be limited to a entity area,
+#' a certain domain,
+#' or on State party nominations for new members.
+#' The function also allows to get information on the
+#' process by which a new member has to go through in
+#' order to accede memberships
+#' @param t A text variable
```

Comment:
Call this argument text to be more explicit? Could otherwise conflict with `t()`?